telco-hub/configuration/reference-crs-kube-compare/required/acm/observabilityMCO.yaml

```diff
@@ -17,16 +17,22 @@ spec:
   # for an scenario with:
   # 3500SNOs, 125 pods and 4 Namespaces (apart from Openshift NS)
   # storage retention 15 days
-  # downsampling disabled
+  # downsampling enabled
   # default MCO Addon configuration samples_per_hour, pv_retention_hrs.
   # More on how to stimate: https://access.redhat.com/articles/7103886
+  # changes on the size affects to the number of replicas, that
+  # affects to cluster capacity and storage
+  # changes on the retention affects to the cluster storage
   advanced:
     retentionConfig:
       blockDuration: 2h
       deleteDelay: 48h
-      retentionInLocal: 24h
-      retentionResolutionRaw: 15d
-  enableDownsampling: false
+      retentionInLocal: {{ .spec.storageConfig.retentionInLocal }}
```

Comment:
I think it is a good opportunity to make this configurable. But keeping the reference with the default values. Users should notice that changes default affect to capacity, cpu, memory and storage.

Reply:
As written this would mean comparing all of these values to spec.storageConfig.retentionResolution which does not (at least in my 4.18 hub) exist. Is that field new or is that a proposal towards ACM for a single field?